src/firebase/userDb.js

```diff
@@ -12,13 +12,13 @@ import property_json from "./data/propertyData";
 import user_json from "./data/userData";
 
 export async function getAllUsers() {
-	const allUsers = "user_mock_data";
-	const querySnapshot = await getDocs(collection(db, allUsers));
-	let users = [];
-	querySnapshot.forEach((doc) => {
-		users.push(doc.data());
-	});
-	return users;
+  const allUsers = "user_mock_data";
+  const querySnapshot = await getDocs(collection(db, allUsers));
```

Comment:
```suggestion
  const querySnapshot = await getDocs(collection(db, "users"));
```
this can be changed back to "users" since we're not uploading to "user_mock_data" anymore

Reply:
resolved earlier :+1:

Reply:
1. Changed the "Not Applicable" to empty string 
2. Some of the contact info are randomly generated. I feel like we can leave it like this for now since the contact infos are fake anyway.

Reply:
sounds good. good to merge on my end!